docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -319,14 +319,14 @@ you should do those steps:
 
 .. code-block:: docker
 
-    #image: ${AIRFLOW_IMAGE_NAME:-apache/airflow:2.6.1}
+    #image: ${AIRFLOW_IMAGE_NAME:-apache/airflow:2.8.0}
     build: .
 
 2) Create ``Dockerfile`` in the same folder your ``docker-compose.yaml`` file is with content similar to:
 
 .. code-block:: docker
 
-    FROM apache/airflow:2.6.1
+    FROM apache/airflow:2.8.0
```

Comment:
to avoid updating it after each release, maybe we can use:
```suggestion
    FROM apache/airflow:${AIRFLOW_VERSION}
```
as we do after two lines.